### DISCUSSION TITLE
Autostart

### DISCUSSION THREAD

**Discussion by QuMix64** [dragonfruitnetwork/onionfruit#54] [Ideas]
Hi, please add the ability to automatically launch the program with Windows and automatically connect to Tor. I think it would be great to minimize interaction with the program

**Comment by aspriddell** [comment]
This will be added along with some other features with the eventual goal to achieve feature parity

**Reply by aspriddell** [reply]
Added in cd32e505290dfe3bef328d89088ec3e49abf83a8, will be available in the next build (over the next couple of days)
